test/fakeintake/client/client.go

```diff
@@ -93,6 +94,18 @@ func (c *Client) getLogs() error {
 	return c.logAggregator.UnmarshallPayloads(payloads)
 }
 
+// getFlare queries the Fake Intake to fetch flare that were sent by a Datadog Agent and returns a Flare struct
+// TODO: handle multiple flares
+func (c *Client) getFlare() (flare.Flare, error) {
+	payloads, err := c.getFakePayloads("/support/flare")
+	if err != nil {
+		return flare.Flare{}, err
+	}
+
+	// TODO: create a flare aggregator and populate flare after parsing it + return an error
+	return flare.ParseRawFlare(payloads[0])
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
You should return the last item here instead of the first one

Reply:
Done in 409f46348d4ca84cc81ab3d0dad136918844fd79